app/models/compute_resource.rb

```diff
@@ -424,9 +424,10 @@ def sanitize_url
     self.url = url.chomp("/") unless url.empty?
   end
 
-  def random_password
+  def random_password(size = 16)
```

Comment:
Can we be explicit in the naming
```suggestion
  def random_password(characters = 16)
```